automation/tests/test_refactor.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from pathlib import Path
+print(Path)
+sys.path.append(str(Path(__file__).parent.parent.absolute()))
+
+import pytest
+import pytest_html
+
+from pages import landingPage
+from pages import basePage
+from pages import spotifyPage
+
+from settings import settings
+
+import time
+
+@pytest.mark.usefixtures("driver_init")
+class BasicTest:
+    pass
+
+class Test_Login(BasicTest):
```

Comment:
Given the login page is external to the main app we probably don't need to write many tests for that. Better to focus on game interactions (I know you'll still need the PO for login page to sign in) you actually want to test :)

Reply:
hadn't realised that was still the name, the test class now covers a larger scope so happy to change it.